notebooks/official/tensorboard/tensorboard_vertex_pipeline.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<p>using KFP SDK -&gt; using the KFP SDK</p><p> </p><p>SDK and execute -&gt; SDK, execute</p><p> </p><p>Setup service account -&gt; Setup a service account | Setup service accounts</p><p> </p><p>customized -&gt; custom</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>